### PR TITLE
[readme] update README file

### PR DESCRIPTION
**Pull request by loganek**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,76 +5,143 @@
...
-### Add to your project's build system
-
-[TBD]
 
 ### Collect the data
 
-[TBD]
+For now HawkTracer provides a simple application for converting event stream to a JSON format which can be consumed by the viewing application: `hawktracer-to-json`. You can use it with a file or with a network stream. Assuming your events have been saved to `file_name.htdump` file, you can generate the JSON file running following command:
+```bash
+hawktracer-to-json --source file_name.htdump --output output_file.json
```
Comment (brunogouveia) [inline README.md]: [Tiny] It would be nice to add an example with network stream.

File: README.md [hunk 2]
```diff
@@ -5,76 +5,143 @@
...
 HawkTracer is a highly portable, low-overhead, configurable profiling tool built in Amazon Video for getting performance metrics from low-end devices.
 
+HawkTracer is available on most popular platforms: **Linux**, **Windows**, **OS X**. The library can be used with **C** and **C++**.
+
+The library provides many different types of events (e.g. CPU usage event, duration tracepoint), but the list can easily be extended by the user.
+
+## Features
+* Low CPU/memory overhead
+* Multi-platform
+* Highly configurable on runtime
```
Comment (andrewrch) [inline README.md]: Any examples of this documented anywhere? I'm not sure what is configurable

File: README.md (hunk 1 above)
Comment (loganek) [reply README.md]: Yeah, do agree. However, I didn't want to create this readme too long. When we have more detailed documentation, we should definitely add a link here to a page where network listeners/streams are described.

File: README.md (hunk 2 above)
Comment (loganek) [reply README.md]: Unfortunately, it's not documented yet. You can configure size of internal buffers, allocators, timelines (which features should be supported). However I might change it to 
`Highly configurable and extensible`
as you can extend this project in a few ways (implement your own features, streams, listeners).
What do you think?

**Comment by loganek** [comment]
The change has been pushed to master branch.

**Pull request closed without merging**